apps/atomy-q/API/tests/Feature/QuoteSubmissionWorkflowTest.php

```
@@ -296,6 +298,114 @@ public function test_quote_submission_upload_without_rfq_lines_does_not_become_r
+            'tenant_id' => $user->tenant_id,
+            'rfq_id' => $rfq->id,
+            'status' => 'uploaded',
+        ]);
+
+        Queue::assertPushed(
+            ProcessQuoteSubmissionJob::class,
+            static function (ProcessQuoteSubmissionJob $job) use ($quoteId): bool {
+                return $job->quoteSubmissionId === $quoteId;
+            }
+        );
+    }
```

PHPMD (2.15.0)

[error] The method test_quote_submission_upload_stays_uploaded_when_queue_default_is_async is not named in camelCase. (undefined)

(CamelCaseMethodName)